lib/rubocop/cop/rspec/indexed_let.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Do not set up test data using indexes (e.g., `item_1`, `item_2`).
+      #
+      # It makers reading the test harder because it's not clear what exactly
+      # is tested by this particular example.
+      #
+      # @example
+      #   # bad
+      #   let(:item_1) { create(:item) }
+      #   let(:item_2) { create(:item) }
```

Comment:
This doesn't reflect the fact that the cop tolerates 3 indexed definitions.

Reply:
Now we have one, so example is valid again